File: Examples/VisualBasic/Load-Save-Convert/ExportToHTML.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports Aspose.Diagram

Namespace Diagrams
    Public Class ExportToHTML
        Public Shared Sub Run()
            ' ExStart:ExportToHTML
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_LoadSaveConvert()
            ' Load diagram
            Dim diagram As New Diagram(dataDir & "ExportToHTML.vsd")
            ' Save diagra in the HTML format
            diagram.Save(dataDir & "ExportToHTML_out.html", SaveFileFormat.HTML)
            ' ExEnd:ExportToHTML
        End Sub
    End Class
End Namespace

File: Examples/CSharp/Load-Save-Convert/ExportToHTML.cs
using System.IO;
using Aspose.Diagram;

namespace Aspose.Diagram.Examples.CSharp.Diagrams
{
    public class ExportToHTML
    {
        public static void Run()
        {
            // ExStart:ExportToHTML
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_LoadSaveConvert();
            // Load diagram
            Diagram diagram = new Diagram(dataDir + "ExportToHTML.vsd");
            // Save diagram
            diagram.Save(dataDir + "outputVSDtoHTML.html", SaveFileFormat.HTML);
            // ExEnd:ExportToHTML
        }
    }
}